patches/relay-mock-network-layer+2.0.0.patch

```diff
@@ -0,0 +1,45 @@
+patch-package
+--- a/node_modules/relay-mock-network-layer/index.js
++++ b/node_modules/relay-mock-network-layer/index.js
+@@ -24,6 +24,7 @@ module.exports = function getNetworkLayer({
+     mocks,
+     resolvers,
+     resolveQueryFromOperation,
++    fieldResolver,
+ 
+     /** https://www.apollographql.com/docs/graphql-tools/mocking.html#Mocking-interfaces */
+     preserveResolvers = false,
+@@ -51,7 +52,9 @@ module.exports = function getNetworkLayer({
+         });
+ 
+         // Add mocks, modifies schema in place
+-        addMockFunctionsToSchema({ schema: executableSchema, mocks, preserveResolvers });
++        if (mocks) {
++            addMockFunctionsToSchema({ schema: executableSchema, mocks, preserveResolvers, defaultFieldResolver: fieldResolver });
```

Comment:
this is where the weird mocking behaviour gets injected, so now we can avoid it by not passing the `mocks` fn

Reply:
@ds300 - should this be PR'd upstream?